bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -33,20 +32,18 @@
 
     @Override
     public synchronized void createPartControl(Composite parent) {
-        try {
-            browser = new Browser(parent, SWT.NONE);
-            browser.addProgressListener(new PitUiUpdatePublisher(browser));
-        } catch (SWTError e) {
-            MessageBox messageBox = new MessageBox(parent.getShell(), SWT.ICON_ERROR | SWT.OK);
-            messageBox.setMessage("Browser cannot be initialized.");
-            messageBox.setText("Exit");
-            messageBox.open();
-        }
+        PitclipseUiUtils.safeExecute
+            (() -> {
```

Comment:
Just cosmetic, why is this in a new line?

Reply:
Just to separate better visually the 3 arguments